docs/dashboards/restricted-operators-dashboards.md

```diff
@@ -0,0 +1,69 @@
+---
+id: restricted-operators-dashboards
+title: Restricted Operators in Dashboards
+sidebar_label: Restricted Operators in Dashboards
+description: Learn more about the restrictions when you use operators with dashboards.
+---
+
+This page has information about some restrictions when you use operators with dashboards.
+
+## Dashboard restrictions
+
+The following operators can't be used with dashboards:
+
+* Details
+* LogReduce
+* LogCompare
+* Parse multi
+* Sample (internal-use operator)
+* Save
+* Transaction
+
+## Auto refresh restrictions
```

Comment:
Was this the part of the UI you are referring to? 

<img width="767" alt="Screenshot 2023-11-01 at 1 12 21 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/70b4a11b-acd8-404d-b55f-95545f3b3bfb">

Reply:
@kimsauce yes.